generator/index.html

```diff
@@ -77,8 +77,10 @@ <h2 id="sprite-name"></h2>
 }
 
 function id_from_url (url) {
-  if (url.indexOf("#") !== -1) url = url.slice(0, url.indexOf("#"));
-  var id = (url + " ").split(/[^0-9]+/)[1];
+  if(url.indexOf("phosphorus") <= url.indexOf("scratch")) {
```

Comment:
This pattern is probably rather elegant, but it's really not immediately obvious. Please could you rewrite to remove the `indexOf` comparison? Thanks :)